File: Week1 C Problems/Mario-less.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("enter number between 1 to 9: ");
    }
    while (h < 1 || h > 8);

    for (int row = 0; row < h; row++)
    {
        for (int space = h - 1; space > row; space--)
        {
            printf(" ");
        }
        for (int hash = 0; hash <= row; hash++)
        {
            printf("#");
        }
        printf("\n");
    }

    return 0;
}

File: Week6 Python Problems/mario-less.py


from cs50 import get_int

# Prompt user for height until it is between 1 and 8
while True:
    height = get_int("Height: ")
    if 1 <= height <= 8:
        break

# Build half-pyramid
for row in range(1, height + 1):
    # Print spaces first
    print(" " * (height - row), end="")
    # Then print hashes
    print("#" * row)
